packages/visual-editor/scripts/generateTranslations.ts

```diff
@@ -0,0 +1,188 @@
+import fs from "fs/promises";
+import path from "path";
+import pLimit from "p-limit";
+
+const defaultLng = "en";
+const ns = "visual-editor";
+const localesDir = "./locales";
+const CONCURRENCY_LIMIT = 5;
+
+const isDryRun = process.argv.includes("--dry-run");
+
+const targetLngs: string[] = [
+  "zh_CN",
```

Comment:
Let's share these between the i18n config and here. Anytime we need to keep things in sync separately it's not great.